python/cudf_polars/cudf_polars/containers/column.py

```diff
@@ -238,6 +244,9 @@ def check_sorted(
             The requested sort order.
         null_order
             Where nulls sort to.
+        stream
+            CUDA stream used for device memory operations and kernel launches
+            on this dataframe.
```

Comment:
All these docstrings should add notes about the importance of `self` (`Column`) being valid on `stream`.